theme/src/components/nav-items.js

```diff
@@ -28,8 +29,9 @@ function NavItem({href, children}) {
 }
 
 function NavItems({items}) {
+  const {containerRef} = useFocusZone({bindKeys: FocusKeys.ArrowVertical | FocusKeys.HomeAndEnd})
```

Comment:
Random question, hope this makes sense 👀 Is there an expectation in a `role="navigation"` context to navigate via a roving tab index? I would have expected this more in an ARIA Menu navigation scheme where-as `role="navigation"` would use tab stops, or maybe the ability to use either scheme.

Separately, when using roving tab, would we need to indicate orientation in some way to communicate that Arrow Up/Down should be used?

Reply:
I've researching about it and I have to say that you're absolutely right. I think I know my mistake. I initially thought about the component as a listbox menu with their own roles, but slowly I was deleting roles, and I totally overseen this use case. Thanks for the review @joshblack :) I will close the PR then.